model-engine/model_engine_server/infra/gateways/gcs_file_storage_gateway.py

```diff
@@ -0,0 +1,104 @@
+import os
+from typing import List, Optional
+
+from google.cloud import storage
+
+from model_engine_server.core.config import infra_config
+from model_engine_server.domain.gateways.file_storage_gateway import (
+    FileMetadata,
+    FileStorageGateway,
+)
+from model_engine_server.infra.gateways.gcs_filesystem_gateway import GCSFilesystemGateway
+
+
+def get_gcs_key(owner: str, file_id: str) -> str:
```

Comment:
nit: I'd prefix these w/ an underscore so that no one is tempted to try and import these from outside this file, thus breaking Clean Architecture norms.

Reply:
btw I think the s3_file_storage_gateway also doesn't have the prefixes